src/trace_processor/dataframe/impl/bytecode_interpreter_test_utils.h

```diff
@@ -75,8 +75,13 @@ struct Fetcher : ValueFetcher {
   uint32_t i = 0;
 };
 
-inline std::string FixNegativeAndDecimal(const std::string& str) {
-  return base::ReplaceAll(base::ReplaceAll(str, ".", "_"), "-", "neg_");
+inline std::string FixNegativeAndDecimalAndDouble(const std::string& str) {
+  std::vector<std::string> replace_with_underscore = {".", "(", ")"};
+  std::string result = str;
+  for (const auto& to_replace : replace_with_underscore) {
+    result = base::ReplaceAll(result, to_replace, "_");
+  }
+  return base::ReplaceAll(result, "-", "neg_");
 }
```

Comment:
at this point it's nicer to have a for loop like:

std::string result = str;
if (results.size() > 0 && result[0] == '-')
  result = "neg_" + result.substr(1)
  for (char& c : result) {
        if (c == '.' || c == '(' || c == ')' || c == '-') {
            c = '_';
        }